tests/tasks/start_mock_wifi.yml

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Ensure NetworkManager is running
+  service:
+    name: NetworkManager
+    state: started
+
+- name: Load mac80211_hwsim kernel module to mock a wifi network
+  shell: modprobe mac80211_hwsim radio=2 && sleep 5
+
+- name: Restart NetworkManager and wpa_supplicant
+  service:
+    name: "{{ item }}"
+    state: restarted
+  with_items:
+    - NetworkManager
+    - wpa_supplicant
+
+- name: Configure wlan0 and wlan1 (mock wifi interfaces)
```

Comment:
This is not configuring `wlan0` only `wlan1`, is that correct?

Reply:
Changed into `Bring up wlan0 and wlan1`